Week2/homework/Exercise1.js

```diff
@@ -0,0 +1,19 @@
+const myNumbers = [1, 2, 3, 4];
+const evenNums = myNumbers.filter(evenFunct);
+const doubleNumbers= evenNums.map(doubleFunct);
+
+function doubleFunct (nums){
+  var ddNum;
+  ddNum=nums*2;
```

Comment:
You should prefer the use of let, const instead of var. Also you can declare and initialise a variable in one line  
eg `var ddNum = nums*2`